e2e/path/main.go

```diff
@@ -104,6 +104,30 @@ var tests = []test{
 		fallback: true,
 		status:   404,
 	},
+	{
+		name: "Fallback when path not given to chained path records",
```

Comment:
Can you elaborate a bit more what your thinking was on that test? Mostly me not being sure about the description.

Reply:
Should've chosen a better description, just a test to make sure requests to a chained path record don't go further when the path is empty and go into fallback instead. Not sure if it's useful, we already have the same test but it's not for a chained record.

Reply:
Ahh so it tries to test it prioritising the rec.To over the further chained records. Sounds good. Just needs a better description.